spec/api/v1/users_controller_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+describe Api::V1::UsersController do
+  describe 'GET /api/users' do
```

Comment:
I would consider adding testcase in which you check what happens if there are no users?